scripts/codespell_whitelist.txt

```diff
@@ -13,3 +13,4 @@ hist
 otion
 keypair
 ether
+arithmetics
```

Comment:
I don't see this word used anywhere in this PR and to me it looks like an actual mistake. Are you sure you want to add it?

Reply:
I added ` --check-filenames` to `codespell`, but maybe this should not be done in this PR. I will remove this.